### PR TITLE
prop-deprecation - required prop - support method and member

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
prop-deprecation - required prop - support method and member

## Changelog
prop-deprecation - required prop - support method and member

### CODE REVIEW

File: eslint-rules/tests/lib/rules/component-prop-deprecation.js [hunk 1]
```diff
@@ -97,6 +97,22 @@ ruleTester.run('component-prop-deprecation', rule, {
...
+<Picker {...getPickerProps()}/>
+`
+    },
+    {
+      options: ruleOptions,
+      code: `
+import {Picker} from 'module-with-deprecations';
+pickerProps = {t: "title", s:"subtitle", migrate: true};
+<Picker {...this.pickerProps}/>
+`
```
Comment (lidord-wix) [inline eslint-rules/tests/lib/rules/component-prop-deprecation.js:114]: Please fix the indentation here :)

**Review by lidord-wix — CHANGES_REQUESTED** [review]
Cool!
Just a small indentation issue :)

File: eslint-rules/tests/lib/rules/component-prop-deprecation.js (hunk 1 above)
Comment (M-i-k-e-l) [reply eslint-rules/tests/lib/rules/component-prop-deprecation.js:114]: Updated automatically, don't think it fixed what you meant, but I'm not really sure which line you wanted me to fix :/

Comment (lidord-wix) [reply eslint-rules/tests/lib/rules/component-prop-deprecation.js:114]: lines 103-106 and 111-114
each line should have an indentation of 8 spaces but have 0

Comment (M-i-k-e-l) [reply eslint-rules/tests/lib/rules/component-prop-deprecation.js]: I don't see any errors...
I'll assume you meant 6 spaces, I've indented the same as the rest of the file.

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**